JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R

```diff
@@ -639,3 +632,11 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
   om <- out$omega
   return(om)
 }
+
+.reverseScoreItems <- function(dataset, options) {
```

Comment:
I *think* this code is the same as the other `.reverseScoreItems()` you defined? You only need one of them. All functions in JASP are available in all files. You could even call any function in `anova.R` file if you wanted to. When you define two functions with the same name one will simply overwrite the other.